File: pythonProject1/53.py
# Take input from the user to make a dictionary and print out the capital of country form the dictionary
d = {}

n = int(input("Enter the number of countries: "))

for i in range(0, n):
    country = input("Enter the name of Country: ")
    capital = input('Enter the capital name: ')

    d[country] = capital

print(d)
e = input("Enter the Country: ")
for i in d:
    if i == e:
        print("The name of the Capital of", e, "is", d[i])

File: C/Programs/53.c
#include<stdio.h>
int main(){
	
int lines, i, j;

printf("Enter the number of lines: ");
scanf("%d", &lines);

for(i=1;i<=lines;i++){
	for(j=1;j<=i;j++){
		printf("%d", j);
	}
	printf("\n");
}

for(i=lines;i>=1;i--){
	for(j=1;j<=i;j++){
		printf("%d", j);
	}
	printf("\n");
}


return 0;
}
